src/components/Icons/InternshipsIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import { SvgIcon, SvgIconProps } from '@material-ui/core';
+import * as React from 'react';
+import { FC } from 'react';
+
+// Can't change svg styling without having it be an svg. Maybe should have a separate folder
+// in source for svg's. Can't import svg's outside of src due to create react app settings
+export const InternshipsIcon: FC<SvgIconProps> = (props) => (
+  <SvgIcon
+    viewBox="0 0 16 17"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <g clipPath="url(#clip0)" strokeWidth=".5">
```

Comment:
These two aren't doing anything.
```suggestion
    <g>
```

Reply:
Removed